src/jupyter_contrib_nbextensions/nbextensions/runtools/main.js

```diff
@@ -1,15 +1,153 @@
 // Add toolbar buttons for extended code execution commands
 
-
 define([
     'base/js/namespace',
     'jquery',
     'require',
     'base/js/events',
+    'services/config',
+    'base/js/utils',
     'codemirror/lib/codemirror'
-],   function(IPython, $, require, events, codemirror) {
+],   function(Jupyter, $, require, events, configmod, utils, CodeMirror) {
     "use strict";
 
+    var base_url = utils.get_body_data("baseUrl");
+    var config = new configmod.ConfigSection('notebook', {base_url: base_url});
```

Comment:
it's better I think to use `Jupyter.notebook.config`, which is loaded by the page already, and means we don't have to make a second request to the server (or load utils/config modules)